gp-includes/formats/format_pomo.php

```diff
@@ -45,6 +51,36 @@ public function read_originals_from_file( $file_name ) {
 		return $this->read_translations_from_file( $file_name );
 	}
 
+	/**
+	 * Create a string that represents the value for the "Language:" header for a po file.
+	 *
+	 * @since 2.1.0
+	 *
+	 * @param GP_Locale $locale The locale object.
+	 *
+	 * @return string|false
+	 */
+	private function generate_language_string( $locale ) {
+		$ret = '';
+
+		if ( $locale->lang_code_iso_639_1 ) {
+			$ret = $locale->lang_code_iso_639_1;
+		} else if ( $locale->lang_code_iso_639_2 ) {
```

Comment:
Should be `elseif`, see https://make.wordpress.org/core/handbook/best-practices/coding-standards/php/#use-elseif-not-else-if.